core/package.json

```diff
@@ -37,6 +37,10 @@
   },
   "devDependencies": {
     "@axe-core/playwright": "^4.7.3",
+    "@capacitor/core": "^5.1.1",
+    "@capacitor/haptics": "^5.0.5",
+    "@capacitor/keyboard": "^5.0.5",
+    "@capacitor/status-bar": "^5.0.5",
```

Comment:
I think it would be good to have dependabot update these deps so we can be aware of any changes to the types

Reply:
https://github.com/ionic-team/ionic-framework/pull/27755/commits/263604cd991e8a69492de0b2e5a964ec2cb641ac